projects/ad713x_fmcz/src/ad713x_fmc.c

```diff
@@ -291,6 +291,29 @@ int main()
 		.dcache_invalidate_range = (void (*)(uint32_t, uint32_t))Xil_DCacheInvalidateRange,
 	};
 
+	struct xil_uart_init_param platform_uart_init_par = {
+#ifdef XPAR_XUARTLITE_NUM_INSTANCES
```

Comment:
Use `PLATFORM_MB` instead

Reply:
taking this back, the current condition is the correct one

Reply:
You could use a UARTLite IP even if you have a PS (it depends on the HDL design and not on the processor used).